_overviews/toolkit/sttp-variable-urls.md

```diff
@@ -0,0 +1,33 @@
+---
+title: How to construct URLs from variables
+type: section
+description: How to construct URLs from variables with Scala Toolkit.
+num: 14
+previous-page: sttp-make-request
+next-page: sttp-request-parameters
+---
+
+{% include markdown.html path="_markdown/install-sttp.md" %}
+
+## URLs from variables
+You will often find yourself in a situation where the address you want to make a request to has some changing parts. 
+For example, `https://example.com/john` may be a page about someone called John, but you want to be able to check out people with different names.
+
+## The `uri` operator
+The `uri` operator (e.g. in `uri"https://example.com/"`) allows you to safely create a web address, also called URI.
+When you have a variable in scope, for instance `name`, you can put its value in the URI like this: `uri"https://example.com/$name"`.
```

Comment:
same as in the section on parameters, I'd mention that this properly escapes any special characters